sinks/ssfmetrics/metrics.go

```diff
@@ -43,33 +44,34 @@ func NewMetricExtractionSink(mw []Processor, timerName string, cl *trace.Client,
 }
 
 // Name returns "metric_extraction".
-func (m *metricExtractionSink) Name() string {
+func (m *MetricExtractionSink) Name() string {
 	return "metric_extraction"
 }
 
 // Start is a no-op.
-func (m *metricExtractionSink) Start(*trace.Client) error {
+func (m *MetricExtractionSink) Start(*trace.Client) error {
 	return nil
 }
 
-func (m *metricExtractionSink) sendMetrics(metrics []samplers.UDPMetric) {
+// SendMetrics enqueues the metrics into the worker channels
+func (m *MetricExtractionSink) SendMetrics(metrics []samplers.UDPMetric) {
 	for _, metric := range metrics {
 		m.workers[metric.Digest%uint32(len(m.workers))].IngestUDP(metric)
 	}
 }
 
-func (m *metricExtractionSink) sendSample(sample *ssf.SSFSample) error {
+func (m *MetricExtractionSink) sendSample(sample *ssf.SSFSample) error {
```

Comment:
I chose not to make this the primary interface because it would result in packaging up a metric in an `SSFSample` only to discard the extra allocation, unwrapping it to convert it into a `UDPMetric`. What do you think about using an interface that takes all components of the metric as separate arguments and the function composes it into a `UDPMetric` behind encapsulation instead of the interface expecting the single metric/sample (and potential extra allocation)?